### ISSUE TITLE
[deleted] Hold alternate character view to select

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#150]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

The system keyboard functionality for hold-to-select keys is that the user must continue to hold and then drags across the character view with the selected character then changing color. The way that hold-to-select in Scribe currently works is that the view is just generated and stays statically until one of the keys is pressed (another key being pressed also clears it).

Now that #145 has finished the current version where the user does not have to hold, discussions could be had about how to change the feature so it's identical to the system functionality. This would then be changed to the default, with the user then being given the option to change to the current no-hold version within the settings that will be made in #16.

- As settings for the app are needed, this issue is blocked by #16 

### Contribution

This issue is not a priority as the current version provides the needed functionality and is apparent as soon as the user tries to drag (the view will still be there once the drag is finished, and then they can select). I'd be willing to work on this with others at some point though :)

**Comment by andrewtavis** [comment]
Deleting this as work will continue in #243 for this 😊
